libs/ledger-live-common/src/families/ethereum/modules/send.ts

```diff
@@ -87,7 +87,11 @@ const send: ModeModule = {
         const feePerGas = EIP1559ShouldBeUsed(a.currency)
           ? t.maxFeePerGas
           : t.gasPrice;
-        amount = a.spendableBalance.minus(gasLimit.times(feePerGas || 0));
+        // Prevents a send max with a negative amount
+        amount = BigNumber.maximum(
+          a.spendableBalance.minus(gasLimit.times(feePerGas || 0)),
+          0
+        );
```

Comment:
This is a classic trap, many other integrations have fallen into it...
It would make a lot of sense to have a standard bridge (dataset) integration test for it, but that's for another time.